lib/ansible/modules/remote_management/manageiq/manageiq_alert_profiles.py

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Red Hat Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+
+module: manageiq_alert_profiles
+
+short_description: Configuration of alert profiles for ManageIQ
+extends_documentation_fragment: manageiq
+version_added: '2.5'
+author: Elad Alfassa (ealfassa@redhat.com)
+description:
+  - The manageiq_alert_profiles module supports listing, adding, updating and deleting alert profiles in ManageIQ.
+
+options:
+  state:
+    description:
+      - absent - alert profile should not exist,
+      - present - alert profile should exist,
+      - list - return a list of alert profiles.
+    choices: ['absent', 'present', 'list']
+    default: 'present'
+  description:
+    description:
+      - The unique alert profile description in ManageIQ.
```

Comment:
I thought it was globally unique, just like with alerts, but apparently alert profiles don't enforce uniqueness of the description. The module *assumes* its globally unique, but this is not necessarily true... I've sent an email to the relevant ManageIQ people to see if this can be changed.

Reply:
So the new plan is to use `name` which is unique.
Creating from ansible, we can control both name and description.
Creating from UI only asks for description; it currently sets name to a GUID, will be changed to equal description.